packages/@react-spectrum/s2/src/Popover.tsx

```diff
@@ -221,6 +238,26 @@ function Popover(props: PopoverProps, ref: DOMRef<HTMLDivElement>) {
     }
   }, [locale, direction, domRef]);
 
+  // On small devices, show a modal (or eventually a tray) instead of a popover.
+  let isMobile = useIsMobileDevice();
```

Comment:
Ah, I had previously commented that my phone was showing the Menu as a modal now instead of a popover but just noticed this. I guess that's ok since we don't truly support mobile fully until we implement trays in S2 but it is a bit weird that the submenus appear as modals. 

However, S2 ComboBox and Picker doesn't even display any sort of dropdown or modal anymore but again, might be ok if our stance is that mobile isn't supported yet. Otherwise, if we want to keep them rendering as popovers, then we might have to do some temp API stuff to preserve that behavior for now.